docs/user_guide/capsules/isolation.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: documentation
+title: Capsule Isolation
+sidebar: sidebars/documentation.html
+---
+
+# Capsule Isolation
+
+The goal of Capsule Isolation is to provide a single endpoint for all of a client's communication, so that in remote network segments, you need only open Firewall ports to the Capsule itself.
```

Comment:
I might reword as `client's communication. Thus, in a remote network segment, you need only open firewall ports to the Capsule itself.`